src/python/pants/engine/target_test.py

```diff
@@ -904,6 +913,10 @@ def hydrate(sources_cls: Type[Sources], sources: Iterable[str]) -> HydratedSourc
 # -----------------------------------------------------------------------------------------------
 
 
+class SmalltalkSources(Sources):
+    pass
```

Comment:
We were previously using `FortranSources`, which does not subclass `Sources` so MyPy was complaining.